### PR TITLE
Add Hacktoberfest page

### PR DESCRIPTION
**Pull request by christian-bromann**

### CODE REVIEW

**Review by diemol — APPROVED** [review]

File: assets/js/hacktoberfest.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+const LOADER_CLASSNAME = 'animated-loader'
+const PARTICIPATING_GH_ORGS = ['selenium', 'appium', 'webdriverio', 'saucelabs']
+const PUBLIC_READ_ONLY_TOKEN = 'a007c6a0a64d3b0e71abd59d8c75aa5000022d19'
```
Comment (diemol) [inline assets/js/hacktoberfest.js:3]: Can this be a secret in the repo? 
I mean, we say that we should not log tokens in the code as a best practice.

**Review by diemol — CHANGES_REQUESTED** [review]

File: assets/js/hacktoberfest.js (hunk 1 above)
Comment (christian-bromann) [reply assets/js/hacktoberfest.js:3]: @diemol it's a static page .. there is no way to keep this secret somewhere where the user has no access to it. Also it is a public token with no access rights to anything private.

Comment (diemol) [reply assets/js/hacktoberfest.js:3]: Ok, fair enough. 
However, it is possible with Hugo
https://github.com/SeleniumHQ/seleniumhq.github.io/blob/f487b14616ebdf1bbdd9c1b8c8c93d8384f9bb0d/docs_source_files/layouts/shortcodes/ghcontributors.html#L22

**Review by diemol — APPROVED** [review]

File: assets/js/hacktoberfest.js (hunk 1 above)
Comment (christian-bromann) [reply assets/js/hacktoberfest.js:3]: Sure, but the token would be still displayed in the script after all.

**Pull request merged**